src/java/detectors/output_ignored_on_resultset_next/OutputIgnoredOnResultSetNext.java

```diff
@@ -0,0 +1,53 @@
+package rules.output_ignored_on_resultset_next;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import javax.sql.DataSource;
+
+class OutputIgnoredOnResultSetNext {
+
+    DataSource dataSource;
+
+    // {fact rule=output-ignored-on-resultset-next@v1.0 defects=1}
+    public long getCountNonCompliant(String getCountSqlQuery) throws SQLException {
+        Connection connection = dataSource.getConnection();
+        PreparedStatement preparedStatement = null;
+        ResultSet resultSet = null;
+        try {
+            preparedStatement = connection.prepareStatement(getCountSqlQuery);
+            resultSet = preparedStatement.executeQuery();
+            // Noncompliant: code does not check if the ResultSet is empty.
+            resultSet.next();
+            return resultSet.getLong(1);
+        } finally {
+            preparedStatement.close();
+            resultSet.close();
+            connection.close();
+        }
+    }
```

Comment:
Why is there no comment on the non-compliant case?

Reply:
Need to debug. Removing offending example for now.